packages/mask/background/services/identity/profile/update.ts

```diff
@@ -90,3 +99,56 @@ export async function attachProfile(
 export function detachProfile(identifier: ProfileIdentifier): Promise<void> {
     return detachProfileDB(identifier)
 }
+
+/**
+ * Set NextID profile to profileDB
+ * */
+
+export async function attachNextIDPersonaToProfile(item: ProfileInformationFromNextID, whoAmI: ECKeyIdentifier) {
+    if (!item.linkedPersona) throw new Error('LinkedPersona Not Found')
+    const personaRecord = {
```

Comment:
```suggestion
    const personaRecord: PersonaRecordDB = {
```

Otherwise, you won't get type check in a good way.